app.js

```diff
@@ -0,0 +1,95 @@
+const express = require("express");
+const cors = require("cors");
+const app = express();
+const bodyParser = require("body-parser");
+const School = require("./School.js");
+
+const port = 3000;
+
+let mySchool = new School()
+
+
+app.use(cors());
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({extended: false}));
+
+const timestamp = () => {
+  let date = new Date()
+  return ` ${date.getFullYear()}, ${date.getMonth() + 1}/${date.getDate()}, ${date.getHours()}:${date.getMinutes()}:${date.getSeconds()}`
+}
+
+const showAllClasses = (req, res) => {
+  res.json({
+   allClasses: mySchool.classes
+  })
+}
+
+const checkIfClassExists = (req, res, next) => {
+  if(mySchool.classes[req.body.name] ){
+  res.json({
+    status: 400,
+    error: "The Class already exist",
+    timestamp: timestamp()
+  })
+}else {
+  next()
+}
+}
+
+const addNewClass =(req, res) => {
+   mySchool.addClass(req.body.name, req.body.teacher)
+  res.json({
+    status: 200,
+    newClass: req.body.name,
+    teacher: req.body.teacher,
+    message: "Created a new class",
+    timestamp: timestamp()
+  })
+}
+
+const enrollNewStudent = (req, res) => {
+  let className = req.params.className;
+  let student = req.body
+  console.log(`"1" ${className}`)
+  console.log(`"2"${student}`)
+  //let enrolledStudent = res.data.enrolledStudent
+  let newStudent = mySchool.enrollStudent(className, student)
+  res.json({
+  status: 200,
+  enrolledStudent: newStudent,
+  className: className,
```

Comment:
This is coming from the frontend though. Stay consistent. It seems you understand how this is working, but take shortcuts by occasionally including your request in your response.